machine_learning/Random Forest/RandomForest.py

```diff
@@ -0,0 +1,520 @@
+"""
+@author: Manish Kumar Chintha
+"""
+
+"""
+list of libraries imported for the program to work.
+"""
+import math
+import numpy as np
+import csv
+from collections import defaultdict
+import operator
+import random
```

Comment:
Please use isort sort order.
```suggestion
import csv
import operator
import math
import random
from collections import defaultdict
from dataclasses import dataclass

import numpy as np
```